apps/full-site-editing/full-site-editing-plugin/full-site-editing/editor/remove-editor-panels/index.js

```diff
@@ -0,0 +1,45 @@
+/* global fullSiteEditing */
+
+/**
+ * External dependencies
+ */
+import { dispatch, select, subscribe } from '@wordpress/data';
+
+/**
+ * Disables specific sidebar editor panels in the FSE context.
+ *
+ * In particular, we remove the featured image panel for pages,
+ * and we remove the post status panel for templates.
+ *
+ * Note that we only need to remove the panel once as it is persisted
+ * in the redux state.
+ */
+const unsubscribe = subscribe( () => {
+	const { getActiveGeneralSidebarName } = select( 'core/edit-post' );
+	const { removeEditorPanel } = dispatch( 'core/edit-post' );
+
+	if ( 'page' === fullSiteEditing.editorPostType ) {
+		removeEditorPanel( 'featured-image' );
+		return unsubscribe();
+	}
+
+	if ( 'wp_template' === fullSiteEditing.editorPostType ) {
```

Comment:
I think we could solve this via CSS only.
The HTML body automatically contains a post type class (e.g. `.post-type-wp_template`) that we could use to only target the Template editor.
So I guess something like this would be enough:
```scss
.post-type-wp_template .components-panel__body.edit-post-post-status {
  display: none;
}
```

Reply:
Haha that's a great point. Much simpler than this. Will update!

Reply:
Updated! Ready for another look.